tests/spec/core/exporter-spec.js

```diff
@@ -28,25 +28,17 @@ describe("Core - exporter", () => {
     expect(doc.querySelectorAll(".removeOnSave").length).toBe(0);
   });
 
-  it("cleans up hyperHTML comments", async () => {
+  it("removes all comments", async () => {
     const ops = makeStandardOps();
-    ops.body = `<div><!---LEAVE%-->PASS<!-- STAY --></div>`;
+    ops.body = `<div><!-- remove -->PASS <span><!-- remove --></span></div>`;
```

Comment:
Previously [we decided not to remove everything](https://github.com/w3c/respec/pull/1514#discussion_r168681074):

>Sometimes Editors add random comments that they don't mind having on TR.

Is it okay now?

Reply:
That's what I was struggling with... I think we just remove them all and see if anyone complains.